payload-schemas/schemas/fork/event.schema.json

```diff
@@ -181,7 +179,8 @@
           },
           "additionalProperties": false
         },
-        "public": { "type": "boolean" }
+        "public": { "type": "boolean" },
+        "organization": { "type": "string" }
```

Comment:
With these changes the `forkee` property matches `repository.schema.json` and the `ref-common-schemas.ts` script replaces it with a reference. Only the `"fork": { "enum": [true] }` difference is significant?

Reply:
These changes are fine.